sdcm/utils/common.py

```diff
@@ -714,6 +715,33 @@ def clean_instances_aws(tags_dict, dry_run=False):
                 LOGGER.debug("Done. Result: %s\n", response['TerminatingInstances'])
 
 
+def remove_test_runner_on_success(runner_ip: Optional[str],
```

Comment:
Consider to look at `clean_sct_runners()` and, probably, extend it to handle this particular case.

Reply:
That was my first choice too, but this would mess with the 'Send email' steps (I believe we're getting the test results using the runner and just cleaning orphaned runners in `clean_sct_runners()`). Discussing this with @bentsi we also wanted to have this be a single responsibility "clean runner when you succeed" step.

Reply:
What do you think about renaming it to `remove_sct_runner_by_ip()`?

Reply:
I decided to extend `clean_sct_runners()` for this, following your original suggestion.